src/lib/create-config.ts

```diff
@@ -111,6 +117,12 @@ const defaultDbOptions: IDBOption = {
             process.env.DATABASE_POOL_IDLE_TIMEOUT_MS,
             secondsToMilliseconds(30),
         ),
+        ...(parseEnvVarBoolean(
```

Comment:
This syntax was new for me, if anyone else doesn't recognise it, this uses the spread operator to optionally assign a property if the boolean condition is truthy. A bit more info here for the curious minded: https://github.com/tc39/proposal-object-rest-spread/issues/45

Reply:
Ooh, that's interesting. Looks right, but maybe it's more legible to do:
```js
...(parseEnvVarBoolean(process.env.ALLOW_NON_STANDARD_DB_DATES, false) ? { afterCreate: dateHandlingCallback} : {})
```

Don't have much of a strong preference either way, though 💁🏼

Reply:
Yeah nice, that does look a lot better, I've swapped that out